pingcap/tidb
# sessionctx: Remove SSLv3 references

### What problem does this PR solve?

This removes references to SSLv3, which as far as I know was never
supported by TiDB or MySQL. As this gets removed from golang this reference
can become a problem.

Issue Number: close #13958

### What is changed and how it works?

What's Changed:

The reference to SSLv3 is removed

When the TLS version isn't found in the map it now returns
'unknown_tls_version' in the status var.

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)

Check the `ssl_version` when connected over TLS. Then comment out the TLS version that is used and try again, which results in 'unknown_tls_version' as expected. 

### Release note 

- No release note
